### PR TITLE
Fix texture build error

### PR DESCRIPTION
**Pull request by xiaowei-guan**
Tizen not support  pixel buffer texture, add empty implementation to fix build error.

### CODE REVIEW

**Comment by swift-kim** [comment]
@xuelian-bai Done? We must at least resolve the header file naming conflict as I said before.

**Comment by xiaowei-guan** [comment]
> @xuelian-bai Done? We must at least resolve the header file naming conflict as I said before.

I have changed file name(shell/platform/tizen/public/flutter_texture_registrar.h -> shell/platform/tizen/public/flutter_tizen_texture_registrar.h).

File: shell/platform/tizen/tizen_embedder_engine.h [hunk 1]
```diff
@@ -20,8 +20,8 @@
 #include "flutter/shell/platform/tizen/channels/text_input_channel.h"
```
Comment (swift-kim) [inline shell/platform/tizen/tizen_embedder_engine.h:20]: Please change the file mode back to 644.

Comment (xiaowei-guan) [reply shell/platform/tizen/tizen_embedder_engine.h:20]: ok

**Review by swift-kim — APPROVED** [review]
Looks good.

**Pull request merged**